src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const newSelectedTabId = selectedTabId
```

Comment:
why do you need this var?

Reply:
Because selectedTabId is const, I can't change it

Reply:
state changes with `setState` if you need and in this component prop `selectedTabId` will change